packages/mobile/src/import/saga.test.ts

```diff
@@ -79,6 +84,29 @@ describe('Import wallet saga', () => {
     await expectSuccessfulSagaWithPhrase(mockValidSpanishPhrase)
   })
 
+  it('initializes account and navigates to home if skipVerification is true', async () => {
```

Comment:
is there also coverage on the case where skipVerification is false (ensuring that the verification screens are shown)?

Reply:
Adding it in 👍